src/org/isf/lab/service/LabIoOperations.java

```diff
@@ -173,25 +147,17 @@
 	 * @return the list of {@link LaboratoryForPrint}s 
 	 * @throws OHServiceException
 	 */
-	public ArrayList<LaboratoryForPrint> getLaboratoryForPrint(
-			String exam, 
-			GregorianCalendar dateFrom, 
-			GregorianCalendar dateTo) throws OHServiceException 
-	{
+	public ArrayList<LaboratoryForPrint> getLaboratoryForPrint(String exam,
+			GregorianCalendar dateFrom,
+			GregorianCalendar dateTo) throws OHServiceException {
 		ArrayList<LaboratoryForPrint> pLaboratory = new ArrayList<LaboratoryForPrint>();
-    	ArrayList<Laboratory> laboritories = null;
-	
-	
-    	if (exam != null) 
-    	{
-    		laboritories = (ArrayList<Laboratory>) repository.findAllWhereDatesAndExamForPrint(dateFrom, dateTo, exam);
-    	}
-    	else
-    	{
-    		laboritories = (ArrayList<Laboratory>) repository.findAllWhereDatesForPrint(dateFrom, dateTo);
-    	}
-		
-		for (Laboratory laboratory : laboritories) 
+    	ArrayList<Laboratory> laboritories = new ArrayList<Laboratory> (
+    			exam != null ?
+								repository.findByExamDateBetweenAndExam_DescriptionContainingOrderByExam_Examtype_DescriptionDesc(dateFrom, dateTo, exam) :
+								repository.findByExamDateBetweenOrderByExam_Examtype_DescriptionDesc(dateFrom, dateTo)
+			);
+
```

Comment:
indentation

Reply:
With #50 the author "added a .editorconfig file, so all editorconfig-aware editors will use tabs from now on.

Reply:
`.editcorconfig` file is only under core project now